FlightJournal.Web/log4net.config

```diff
@@ -0,0 +1,43 @@
+﻿<log4net>
```

Comment:
as mentioned above, i would prefer the config section to be included in web.config so it can remain a clean transformation per environment with different settings.